### ISSUE TITLE
Preserve logfmt-style data

### ISSUE THREAD

**Issue by aseemk** [goodeggs/heroku-log-normalizer#11]
As mentioned in <https://github.com/goodeggs/bites/issues/20#issuecomment-143237021>, it'd be great to preserve the structure of e.g. Heroku router logs:

```
at=info method=GET path="/" host=corp-myapp.herokuapp.com request_id=deaa77fa-c9c8-4eed-a8d2-867ee4e85317 fwd="107.127.127.127,199.127.127.127" dyno=web.2 connect=2ms service=39ms status=200 bytes=745586
```

Logfmt parsing module from Heroku dev:

https://github.com/csquared/node-logfmt

However, as the [parsing](https://github.com/csquared/node-logfmt#parsing) section notes, numeric values are *not* parsed to numbers. So `"bytes": "745586"` would get sent, not `"bytes": 745586`.

But that might just have to be a limitation of this. I'll file an issue on the logfmt module that perhaps it could still attempt to parse numbers, and only fallback to string if precision loss was detected.

WDYT?

**Comment by aseemk** [comment]
https://github.com/csquared/node-logfmt/issues/33

**Comment by bobzoller** [comment]
yeah we did this at one point.  you can probably find it in the git
history.  as long as the solution accounts for unstructured log lines
(!json && !logfmt), I'd be happy to accept a PR.
